### PR TITLE
Repair buildOnHover parameter, remove hoverOnTap

### PR DESCRIPTION
**Pull request by enricostrijks**
Make the buildOnHover parameter work again and remove hoverOnTap since it's not used anywhere else in the code.

### CODE REVIEW

File: lib/src/marker_widget.dart [hunk 1]
```diff
@@ -6,23 +6,22 @@ class MarkerWidget extends StatelessWidget {
...
     required this.marker,
     required this.onTap,
     this.onHover,
     this.buildOnHover = false,
-    this.hoverOnTap,
   }) : super(key: marker.key ?? ObjectKey(marker.marker));
 
   @override
   Widget build(BuildContext context) {
+    print(buildOnHover);
```
Comment (juliaen-dr) [inline lib/src/marker_widget.dart:19]: I think we should remove this?

**Comment by juliaen-dr** [comment]
#174

**Review by juliaen-dr — APPROVED** [review]

**Comment by enricostrijks** [comment]
Thanks, only had 4 hours of sleep so missed that. It should be fixed by now. @lpongetti needs to merge it for it to work so we should wait on that.

**Review by juliaen-dr — APPROVED** [review]

**Comment by juliaen-dr** [comment]
@lpongetti

**Pull request merged**